R/render-plot.R

```diff
@@ -36,12 +36,13 @@
 #' @param res Resolution of resulting plot, in pixels per inch. This value is
 #'   passed to [grDevices::png()]. Note that this affects the resolution of PNG
 #'   rendering in R; it won't change the actual ppi of the browser.
-#' @param alt Alternate text for the HTML `<img>` tag
-#'   if it cannot be displayed or viewed (i.e., the user uses a screen reader).
-#'   In addition to a character string, the value may be a reactive expression
-#'   (or a function referencing reactive values) that returns a character string.
-#'   NULL or "" is not recommended because those should be limited to decorative images
-#'   (the default is "Plot object").
+#' @param alt Alternate text for the HTML `<img>` tag if it cannot be displayed
+#'   or viewed (i.e., the user uses a screen reader). In addition to a character
+#'   string, the value may be a reactive expression (or a function referencing
+#'   reactive values) that returns a character string. By default,
+#'   `ggplot2::get_alt_text()` is used to get a sensible default for `ggplot2`
+#'   plots (other plots default to `"Plot Object"`). `NULL` or `""` is not
+#'   recommended because those should be limited to decorative images.
```

Comment:
The logic of the ggplot2 part is a bit hard to follow. Also, it would be good to explicitly explain the `NA` default value. Maybe something like this:

> If the value is `NA` (the default), then for ggplot2 plots, it will try to use `ggplot2::get_alt_text()` to extract alt text from the ggplot object; for other plots, `NA` will result in alt text "Plot object".